Runtime_Support/dot_net_core.md

```diff
@@ -0,0 +1,25 @@
+# .NET Core on App Service
+
+## .NET Core Update Policy
+
+App Service upgrades the underlying .NET Core runtime and SDK of your application as part of the regular platform updates. As a result of this update process, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
+
+### End of Life
+
+Once a version of .NET Core has reached it's end of life (EOL) it will no longer be available from Runtime Stack selection dropdown.
```

Comment:
Perhaps put the support policy link here around the ```.NET Core has reached it's end of life (EOL)```

Reply:
@timheuer i'm trying to stay consistent with what i'm doing for other run-time stacks:

- [PHP](https://github.com/Azure/app-service-linux-docs/blob/master/Runtime_Support/php_support.md)
- [Python](https://github.com/Azure/app-service-linux-docs/blob/master/Runtime_Support/python_support.md)